sdk/storage/azblob/blockblob/models.go

```diff
@@ -190,15 +194,20 @@ type CommitBlockListOptions struct {
 	RequestID                    *string
 	Tier                         *blob.AccessTier
 	Timeout                      *int32
-	TransactionalContentCRC64    []byte
-	TransactionalContentMD5      []byte
 	HTTPHeaders                  *blob.HTTPHeaders
 	CPKInfo                      *blob.CPKInfo
 	CPKScopeInfo                 *blob.CPKScopeInfo
 	AccessConditions             *blob.AccessConditions
 	LegalHold                    *bool
 	ImmutabilityPolicyMode       *blob.ImmutabilityPolicySetting
 	ImmutabilityPolicyExpiryTime *time.Time
+
+	TransactionalValidation blob.TransferValidationType
```

Comment:
In `blockblob.CommitBlockList`, I see that `TransactionalValidation` is only used for returning error as below.
```
// If user attempts to pass in their own checksum, errors out.
if options.TransactionalValidation != nil && reflect.TypeOf(options.TransactionalValidation).Kind() != reflect.Func {
	return CommitBlockListResponse{}, bloberror.UnsupportedChecksum
}
```
If `TransactionalValidation` is not used in anywhere else in `CommitBlockList` method, we should not add this field. Rather we can have the above check for returning error on `TransactionalContentCRC64` and `TransactionalContentMD5` fields.

Reply:
Thanks for catching this, CommitBlockList is updated accordingly!